cms/src/main/resources/api-specification/codegen-templates/params_table.mustache

```diff
@@ -0,0 +1,12 @@
+<h{{variable "paramsHeadingLevel"}} class="nhsd-t-heading-m">{{variable "paramsHeading"}}</h{{variable "paramsHeadingLevel"}}>
```

Comment:
i see that this is hardcoded as a `nhsd-t-heading-m` heading also, is that intentional?

Reply:
Another good spot! :+1: 

No, it was not, I'm afraid.

I've now moved the whole heading line out of this template and into the caller one (`index.mustache`), applying correct levels and classes there. At first I've introduced yet another variable for the CSS classes to be set before invoking this template, but this increased the boilerplate code to total of three variables, all targeting just the heading line; seeing how excessive this seemed, I got rid of them all and just moved the line outside of the template.